JSDemos/Demos/Charts/PieWithCustomStyles/description.md

```diff
@@ -0,0 +1,47 @@
+This demo shows how to implement custom patterns and gradients in the PieChart component points.
+
+## Gradient
+
+Call the [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to obtain a gradient id. Specify method arguments:
+
+- `type`    
+This method supports two gradient types: *'linear'* and *'radial'*.
+
+- `options`    
+An object with gradient options.
+
+    - `colors`    
+    An array of gradient colors. Each color is an object that includes `color` and `offset` fields. The `offset` field specifies a starting point for each color.
+
+    - `rotationAngle` (linear gradients only)
+    You can use this option to rotate linear gradients.
+
+## Pattern
```

Comment:
Maybe we need to add a sentence in the beginning that makes the connection between this section and what they see in the control: 

```suggestion
## Pattern

This demo uses patterns for two pie slices: Grid and Stripes. 
```